metadata-db-app/src/cmr/metadata_db/data/oracle/concepts.clj

```diff
@@ -380,17 +397,20 @@
                                                                                      (order-by (desc :revision-id)))))))))
          dynamo-concept-get (when (not= "dynamo-off" (dynamo-config/dynamo-toggle))
                               (util/time-execution
-                               ))]
+                               (when-let [get-result (dynamo/get-concept concept-id)]
+                                 (db-result->concept-map-dynamo concept-type (:provider-id provider) get-result))))]
      (when (not= "dynamo-off" (dynamo-config/dynamo-toggle))
-       (info "Runtime of EFS get-concept: " (first efs-concept-get) " ms.\nValue gotten from EFS get-concept: " (second efs-concept-get))
-       (info "Runtime of DynamoDB get-concept: " (first dynamo-concept-get) " ms.\nValue gotten from DynamoDB get-concept: " (second dynamo-concept-get)))
+       (info "Runtime of EFS get-concept: " (first efs-concept-get) " ms.")
```

Comment:
the effort of this ticket is to provide statistics right? Would it help if all the logs had a common prefix so that you can find all of these (and only these) in splunk?

Reply:
I can search for strictly these logs in splunk reliably, but I can go ahead and add a prefix for sure